Apps/Admin/Tests/Functional/cypress/integration/e2e/delegation.cy.js

```diff
@@ -133,9 +133,21 @@ describe("Delegation Protect", () => {
         // Confirmation dialog confirmation button
         cy.get("[data-testid=confirm-button]").should(
             "be.visible",
-            "be.enabled"
+            "not.be.enabled"
```

Comment:
@BronzBierd this change to expect the confirm button to not be enabled is actually about the 'required' nature of the protect reason. Beyond that in every case where the test continues, it has to have text in the box for the confirm button to be enabled and clicked.